src/components/Pagination/PaginationLink.js

```diff
@@ -1,8 +1,22 @@
 import React from 'react';
 
+import {isEqual} from 'lodash';
+
 class PaginationLink extends React.Component {
+  componentWillMount() {
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  shouldComponentUpdate(nextProps) {
+    return !isEqual(this.props, nextProps);
```

Comment:
This is where shouldComponentUpdate is tricky, multiple cases to take into account:
- url changes? (new search = different url)
- active page (click = item class is "active")

So you would end up with a pretty long if .. condition while just comparing all the props deeply is ok (small objects).

But yes, my first iteration was to just do this.props.pageNumber !== nextProps.pageNumber, then it failed